Bottom: f8419c0a0b5e51ea83afebc665ff6919d49ddcde
Top:    c8e891253268d227fac6e057e44f36a126494b72
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2016-01-06 23:12:37 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
